api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -36,10 +37,16 @@
 
   private static final Logger logger = Logger.getLogger(BigQueryService.class.getName());
 
-  @Autowired private BigQuery bigquery;
-
   @Autowired private Provider<WorkbenchConfig> workbenchConfigProvider;
 
+  private BigQuery getBigQueryService() {
+    CdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
```

Comment:
if you're not testing `cdrVersion` then there's no harm in doing this inline below.